src/frontends/pytorch/src/op/contains.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/convert.hpp"
+#include "openvino/op/exp.hpp"
+#include "openvino/op/log.hpp"
+#include "openvino/op/maximum.hpp"
+#include "openvino/op/subtract.hpp"
+#include "openvino/op/broadcast.hpp"
+#include "openvino/op/constant.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_contains(const NodeContext& context) {
+    // "aten::__contains__(Tensor self, Scalar value) -> bool"
+    num_inputs_check(context, 2, 2);
+
+    auto input_tensor = context.get_input(0);
+    auto scalar_value = context.get_input(1);
+
+    scalar_value = context.mark_node(std::make_shared<v0::Convert>(scalar_value, input_tensor.get_element_type()));
+
+    auto broadcast_scalar = context.mark_node(std::make_shared<v0::Broadcast>(scalar_value, input_tensor.get_shape()));
+
+    auto comparison_result = context.mark_node(std::make_shared<v0::Equal>(input_tensor, broadcast_scalar));
+
+    auto reduction = context.mark_node(std::make_shared<v0::ReduceSum>(comparison_result, {0}, true));
```

Comment:
you need to create `reduce_axis` constant. Without it, the code does not work and is not buildable